configure.ac

```diff
@@ -42,199 +38,222 @@ AC_ARG_ENABLE([large-tiles], AS_HELP_STRING([--enable-large-tiles],
+                [[#include <CoreText/CoreText.h>]]
+                ,[[CTFontDescriptorCopyAttribute(NULL, kCTFontURLAttribute);]]
+            )
+        ],[
+            LIBS="$LIBS -framework CoreText -framework CoreFoundation"
+            AC_DEFINE(CONFIG_CORETEXT, 1, [found CoreText framework])
+            coretext=true
+            AC_MSG_RESULT([yes])
+        ],[
+            coretext=false
+            AC_MSG_RESULT([no])
+        ])
+    ])
+])
 
 ### DirectWrite
-if test x$enable_directwrite != xno; then
-# Linking to DirectWrite directly only works from Windows
-AC_MSG_CHECKING([for DIRECTWRITE])
-AC_LINK_IFELSE([
-  AC_LANG_PROGRAM(
-    [[#include <windows.h>]],
-    [[;]],)
-  ], [
-    AC_DEFINE(CONFIG_DIRECTWRITE, 1, [found DirectWrite])
-    directwrite=true
-    AC_MSG_RESULT([yes])
-  ], [
-    directwrite=false
-    AC_MSG_RESULT([no])
-  ])
-fi
+AS_IF([test "x$enable_directwrite" != xno], [
+    # Linking to DirectWrite directly only works from Windows
+    AC_MSG_CHECKING([for DIRECTWRITE])
+    AC_LINK_IFELSE([
+        AC_LANG_PROGRAM(
+            [[#include <windows.h>]]
+            ,[[;]]
+            ,
+        )
+    ],[
+        AC_DEFINE(CONFIG_DIRECTWRITE, 1, [found DirectWrite])
+        directwrite=true
+        AC_MSG_RESULT([yes])
+    ],[
+        directwrite=false
+        AC_MSG_RESULT([no])
+    ])
+])
 
 ## Require at least one system font provider by default
-if test x$enable_require_system_font_provider != xno &&
-   test x$fontconfig != xtrue &&
-   test x$directwrite != xtrue &&
-   test x$coretext != xtrue
-then
-    AC_MSG_ERROR([\
-Either DirectWrite (on Windows), CoreText (on OSX), or Fontconfig \
-(Linux, other) is required. If you really want to compile without \
-a system font provider, add --disable-require-system-font-provider])
-fi
+AS_IF([test "x$enable_require_system_font_provider" != xno  dnl
+        && test "x$fontconfig" != xtrue                     dnl
+        && test "x$directwrite" != xtrue                    dnl
+        && test "x$coretext" != xtrue                       ], [
+    AC_MSG_ERROR(m4_text_wrap(m4_normalize([
+            Either DirectWrite (on Windows), CoreText (on OSX), or Fontconfig
+            (Linux, other) is required. If you really want to compile without
+            a system font provider, add --disable-require-system-font-provider]),
+        [                  ],
+        [No system font provider!],
+        [78]
+    ))
+])
 
 ## Now add packages to pkg_requires
 pkg_requires="freetype2 >= 9.10.3"
 pkg_requires="fribidi >= 0.19.0, ${pkg_requires}"
 pkg_requires="harfbuzz >= 1.2.3, ${pkg_requires}"
-if test x$fontconfig = xtrue; then
+AS_IF([test "x$fontconfig" = xtrue], [
     pkg_requires="fontconfig >= 2.10.92, ${pkg_requires}"
-fi
+])
 
 
 # Locate and configure Assembler appropriately
-AS_IF([test x$enable_asm != xno], [
+AS_IF([test "x$enable_asm" != xno], [
     AS_CASE([$host],
-        [i?86-*], [
+        [i?86-*],[
```

Comment:
I don’t entirely understand where/why you choose to have spaces between arguments and where to omit them.

Reply:
tbh, I don't exactly remember either.
 I think, it might started out with spaces everywhere, but perhaps I thought the comma is harder to miss without a space, or because a sole `    ] , [` in a line *(eg in `AS_IF` with an else clause)* imo looks better without spaces, I started to remove spaces in flow-control macros but wasn't consistent about it?

Reply:
Changed singe `],[` to `], [` and cases now use a separate line for closing square brackets.